bin/citgm-all.js

```diff
@@ -47,6 +56,17 @@ if (!app.su) {
   log.warn('root', 'Running as root! Use caution!');
 }
 
+if (app.withTags){
+  app.withTags = app.withTags.split(',');
+  log.info('withTags', 'Only running tests matching these tags: '
+      + app.withTags);
```

Comment:
This just logs a comma separated output such as:
`Only running tests matching these tags: tag1, tag2`